airbyte-integrations/connectors/source-adaptive-insights/source_adaptive_insights/source.py

```diff
@@ -55,23 +109,209 @@ def request_body_data(
+        <call method="{self.method}" callerName="Airbyte - auto">
+        <credentials login="{self.username}" password="{self.password}"/>
+        <include versionName="Current LBE" inaccessibleValues="true"/>
+        </call>
+        """.encode("utf-8")
+
+        return body
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        json_response = xmltodict.parse(response.content)
+        
+        yield from handle_export_levels(json_response)
+
+
+class ExportAccounts(AdaptiveInsightsStream):
+
+    primary_key = "id"
+    http_method = "POST"
+    method = "exportAccounts"
+
+    def path(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+
+        return "v32"
+
+    def request_body_data(
+        self,
+        stream_state: Mapping[str, Any],
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Optional[Union[Mapping, str]]:
+
+        body = f"""<?xml version='1.0' encoding='UTF-8'?>
+        <call method="{self.method}" callerName="Airbyte - auto">
+        <credentials login="{self.username}" password="{self.password}"/>
+        <include versionName="Current LBE" inaccessibleValues="true"/>
+        </call>
+        """.encode("utf-8")
+
+        return body
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        json_response = xmltodict.parse(response.content)
+        
+        yield from handle_export_accounts(json_response)
+
+
+class ExportData(AdaptiveInsightsStream):
 
     primary_key = "id"
     http_method = "POST"
-    method = "exportDimension"
+    method = "exportData"
 
     def path(
         self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
     ) -> str:
 
         return "v32"
 
+    def construct_account_payload(self) -> str:
+        if not self.accounts:
+            return ""
+        list_of_accounts = self.accounts.split(',')
+        payload = "<accounts>"
+        for account in list_of_accounts:
+            payload += f"<account code=\"{account}\" isAssumption=\"false\" includeDescendants=\"true\"/>"
+        payload += "</accounts>"
+
+        return payload
+
+
+    def construct_xml_body(self, start_date: str, end_date: str) -> str:
+
+        body = f"""<?xml version='1.0' encoding='UTF-8'?>
+        <call method="{self.method}" callerName="Airbyte - auto">
+        <credentials login="{self.username}" password="{self.password}"/>
+        <version name="{self.version_type}" isDefault="false"/>
+        <format useInternalCodes="true" includeCodes="false" includeNames="true" displayNameEnabled="true"/>
+        <filters>
+        {self.construct_account_payload()}
+        <timeSpan start="{start_date}" end="{end_date}"/>
+        </filters>
+        <dimensions>
```

Comment:
Here you have defined the required Dimensions. Is that working with the defined schema?